i18n/locales/en_us.json

```
@@ -64,7 +65,7 @@
   "resizeSettingSectionAnchor": "Anchor",
   "resizeSettingSectionDefaultTitle": "Default size", 
   "resizeSettingSectionDefaultSubmitButton": "Set Default",
-  "resizeSettingSectionDefaultWidth": "Width",
+  "resizeSettingSectionDefaultWidth": "Width", 
```

random whitespace added
```suggestion
  "resizeSettingSectionDefaultWidth": "Width",
```